Versionsmigration/5.4-6.0/xplangml-migration-54-60b.halex.alignment.xml

```diff
@@ -12164,21 +12164,35 @@ if (refLegende){&#13;
         <parameter value="true" name="ignoreNamespaces"/>
         <parameter value="true" name="structuralRename"/>
     </cell>
-    <cell relation="eu.esdihumboldt.hale.align.formattedstring" id="C136c3e16-8af3-4c3d-9ce6-89c660b0af55" priority="normal">
+    <cell relation="eu.esdihumboldt.cst.functions.groovy" id="C136c3e16-8af3-4c3d-9ce6-89c660b0af55" priority="normal">
         <source name="var">
             <property>
                 <type name="XP_RasterdarstellungType" ns="http://www.xplanung.de/xplangml/5/4"/>
                 <child name="id" ns="http://www.opengis.net/gml/3.2"/>
             </property>
         </source>
-        <target>
+        <source name="var">
+            <property>
+                <type name="XP_RasterdarstellungType" ns="http://www.xplanung.de/xplangml/5/4"/>
+                <child name="refText" ns="http://www.xplanung.de/xplangml/5/4"/>
+            </property>
+        </source>
+        <target name="result">
             <property>
                 <type name="XP_PlanType" ns="http://www.xplanung.de/xplangml/6/0"/>
                 <child name="texte" ns="http://www.xplanung.de/xplangml/6/0"/>
-                <child name="href" ns="http://www.w3.org/1999/xlink"/>
             </property>
         </target>
-        <parameter value="#{id}" name="pattern"/>
+        <complexParameter name="script">
+            <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
+if(refText.p.first()){&#13;
+	_target {&#13;
+	href( id )&#13;
+}&#13;
+}
+</core:text>
+        </complexParameter>
+        <parameter value="true" name="variablesAsInstances"/>
```

Comment:
IMO this groovy script here on attribute level is not needed. The additional condition on type level is sufficient.

Reply:
So can it stay formattedstring then? Do I have to adapt anythin or can you merge with changes?

Reply:
Yes, it can stay a formatted string. But you would need to revert this changes before merging as we want to keep it as formatted string.

Reply:
OK, done.